spring-boot-rest-2/src/main/java/com/baeldung/unsupportedmediatype/User.java

```diff
@@ -0,0 +1,66 @@
+package com.baeldung.unsupportedmediatype;
+
+import javax.xml.bind.annotation.XmlRootElement;
+import java.io.Serializable;
+
+@XmlRootElement
+public class User implements Serializable {
+    private Integer id;
+    private String name;
+    private Integer age;
+    private String address;
+
+    public User(){
+    }
+
+    public User(Integer id, String name, Integer age, String address) {
+        this.id = id;
+        this.name = name;
+        this.age = age;
+        this.address = address;
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public Integer getAge() {
+        return age;
+    }
+
+    public void setAge(Integer age) {
+        this.age = age;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    @Override
+    public String toString() {
+        return  "User{"
+              + "id=" + id
+              + ", name='" + name + '\''
+              + ", age=" + age
+              + ", address='" + address + '\''
+              + '}';
```

Comment:
The rule is to indent continuation lines 2 spaces over from where the previous statement began. In this case, it would be 2 spaces to the right of the "return" (under the "t").